cmd/server/main.go

```diff
@@ -15,7 +15,7 @@ func main() {
 		_, _ = w.Write([]byte("pong"))
 	}).Methods(http.MethodGet)
 	r.HandleFunc("/auth", auth.HandleAuth).Methods(http.MethodPost)
-	r.HandleFunc("/requestbody", requestbody.HandleRequestBody).Methods(http.MethodPost)
+	r.HandleFunc("/requestbody/{*}", requestbody.HandleRequestBody).Methods(http.MethodPost)
```

Comment:
don't really need this as in the openapi document you can use paths like `/requestBody#arrayOfArrays` to use the same path for different operations, you can see this in some of the auth paths I setup

Reply:
Ohh gotcha, thanks I'll do that. I did notice the `#` syntax you used earlier and did a quick search on the openapi docs for it, but didn't find anything at the time